doc/strace.1.in

```diff
@@ -57,6 +57,7 @@ strace \- trace system calls and signals
 .OP \-U columns
 .OP \-X format
 .OP \-\-seccomp\-bpf
+.if '@ENABLE_STACKTRACE_FALSE@'#' .OP \-\-stacktrace\-frames\-limit limit
```

Comment:
Using italics for the parameter, `limit` is better:
```
.if '@ENABLE_STACKTRACE_FALSE@'#' .OP \-\-stacktrace\-frames\-limit\fR=\fIlimit\fR
```

Reply:
Done